primitives/README.md

```diff
@@ -1,2 +1,40 @@
 # Valiu Liquidity Network - Primitives
 
+This module defines the basic primitives that form the basis of VLN.
+
+## Asset
+
+An asset is meant to represent the synthetic version of any fiat/crypto currency that is allowed onchain.
+
+```rust
+pub enum Asset {
+    Collateral(Collateral),
+    Fiat(Fiat),
+    Usdv,
+}
+```
+
+#### Collateral
+
+Collateral lists all the cryptocurrencies that can be deposited to create assets onchain.
+
+```rust
+pub enum Collateral {
+        Usdc = "USDC",
```

Comment:
Did we remove BTC? Also how will we recognise a deposit of USDC? 🤔

Reply:
For now yes, it was there for demo purposes but is currently not needed. The USDv in this moment exists as some USDC in an external account that Valiu controls, Valiu as a trusted liquidity provider will "attest" it has USDC as a foreign asset and use that to mint USDv. Is up to Valiu to do this manually or automate it based on how its USDC stock changes.